File: SyncKit/Classes/CoreData/QSPendingRelationship+CoreDataProperties.m
//
//  QSPendingRelationship+CoreDataProperties.m
//  Pods
//
//  Created by Manuel Entrena on 22/10/2016.
//
//

#import "QSPendingRelationship+CoreDataProperties.h"

@implementation QSPendingRelationship (CoreDataProperties)

+ (NSFetchRequest *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"QSPendingRelationship"];
}

@dynamic relationshipName;
@dynamic targetIdentifier;
@dynamic forEntity;

@end

File: SyncKit/Classes/CoreData/QSPendingRelationship+CoreDataProperties.swift
//
//  QSPendingRelationship+CoreDataProperties.swift
//  
//
//  Created by Manuel Entrena on 02/06/2019.
//
//

import Foundation
import CoreData


extension QSPendingRelationship {

    @nonobjc class func fetchRequest() -> NSFetchRequest<QSPendingRelationship> {
        return NSFetchRequest<QSPendingRelationship>(entityName: "QSPendingRelationship")
    }

    @NSManaged var relationshipName: String?
    @NSManaged var targetIdentifier: String?
    @NSManaged var forEntity: QSSyncedEntity?

}
